test/tick-processor/test-tick-processor-polyfill-brokenfile.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+const common = require('../common');
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+if (!common.enoughTestCpu)
+  common.skip('test is CPU-intensive');
+
+if (common.isWindows ||
+  common.isSunOS ||
+  common.isAIX ||
+  common.isLinuxPPCBE ||
+  common.isFreeBSD)
+  common.skip('C++ symbols are not mapped for this os.');
```

Comment:
I'm not sure whether the linter (`make lint`) will complain or not, but this is difficult to read. Please use braces here.